scripts/Competition_Final_get_perf.sh

```diff
@@ -83,7 +83,7 @@ func TestReadWrite(t *testing.T) {
 	}
 }
 '
-
+go clean -testcache
 echo "$bench_file" > kv/test_raftstore/bench_test.go
 go test ./kv/test_raftstore/ -run ReadWrite -v > bench.log
```

Comment:
Adding `-count=1` will also force the test to run. I think that's more lightweight than clearing all the test cache.
```suggestion
go test ./kv/test_raftstore/ -run ReadWrite -v -count=1 > bench.log
```

Reply:
Thank you for your supplement. Yes, this approach is fine too. We have tried Plan A, Plan B, and A+B, and the performance results obtained are similar. Perhaps your method is more lightweight, but as a scoring performance test, we think starting from the beginning is also reasonable.

Reply:
Since it’s a standalone script with only one test to run, I’m fine with the clean-cache approach. If it were part of a larger suite, we’d likely want to avoid affecting unrelated tests by using `count=1`.